CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * Jsdoc `@demo` annotations are now added to `demos` collection for `Element`, `ElementMixin`, `PolymerElement` and `Behavior`.
 * Types and descriptions are now extracted from method @param and @returns jsdoc annotations.
 * Fixed caching issue such that Documents would not always have information from the latest versions of their dependencies.
+* The warning printer displays the squiggle underline in the correct place on lines indented by tabs.
```

Comment:
We stamped alpha.35 just now. Please uncomment Unreleased above and put this bullet point under it.